icons/desktop/dock-side-right-16.svg

```diff
@@ -1,4 +1,4 @@
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-<svg viewBox="0 0 16 16" width="16" height="16" xmlns="http://www.w3.org/2000/svg" fill="context-fill #0b0b0b"><path d="M0 4a3 3 0 0 1 3-3h10a3 3 0 0 1 3 3v8a3 3 0 0 1-3 3H3a3 3 0 0 1-3-3zm2 0v8a1 1 0 0 0 1 1h7V3H3a1 1 0 0 0-1 1zm9-1v10h2a1 1 0 0 0 1-1V4a1 1 0 0 0-1-1z"/></svg>
\ No newline at end of file
+<svg viewBox="0 0 16 16" width="16" height="16" xmlns="http://www.w3.org/2000/svg" fill="context-fill context-fill"><path d="M0 4a3 3 0 0 1 3-3h10a3 3 0 0 1 3 3v8a3 3 0 0 1-3 3H3a3 3 0 0 1-3-3zm2 0v8a1 1 0 0 0 1 1h7V3H3a1 1 0 0 0-1 1zm9-1v10h2a1 1 0 0 0 1-1V4a1 1 0 0 0-1-1z"/></svg>
```

Comment:
Same for this one.

Reply:
There were a few more like that, so I've fixed them all.